### PR TITLE
[cling-cpt] Added skip-cleanup flags for check-requirements and create-dev-env [skip-ci]

### PR DESCRIPTION
**Pull request by saisoma123**
# This Pull request: Set skip-cleanup flags to true for check-requirements and create-dev-env

## Checklist:

- [X] tested changes locally
- [NA] updated the docs (if necessary)

This PR fixes issue in meta-issue list #406 (https://github.com/root-project/cling/issues/406)

### CODE REVIEW

File: interpreter/cling/tools/packaging/cpt.py [hunk 1]
```diff
@@ -1975,6 +1975,7 @@ def custom_input(prompt, always_yes=False):
     tar_required = True
 
 if args['check_requirements']:
+    args['skip_cleanup'] = True
```
Comment (vgvassilev) [inline interpreter/cling/tools/packaging/cpt.py:1978]: Why do we need the `skip-cleanup` when checking for the requirements?

Comment (saisoma123) [reply interpreter/cling/tools/packaging/cpt.py:1978]: @vgvassilev my bad I think I misread your email. I assumed you meant add skip-cleanup to the check_requirements flag too.

Comment (vgvassilev) [reply interpreter/cling/tools/packaging/cpt.py:1978]: Ah, no, only to the other one.

Comment (saisoma123) [reply interpreter/cling/tools/packaging/cpt.py]: Yeah just removed @vgvassilev my bad

**Review by vgvassilev — APPROVED** [review]

**Pull request merged**